JoomGallery/JoomGallery
# Use Bootstrap button and modal in Thumbnail form field

The Thumbnail form field is used in the backend and frontend category edit views to assign a thumbnail to a category.

This pull request changes the simple text link (Select) to open the modal thumbnail selection into a nice Bootstrap button and makes use of the Bootstrap modal box instead of the Mootools one.
It also removes the redundant Thumbnail form field for the frontend.
#### Test instructions

To test this pull request just edit a category in the backend **and** frontend and try to assign a thumbnail to it. Also try to delete an assigned thumbnail again. Check if everything is working like before.
